aws/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -1,6 +1,6 @@
 # Self-hosted Kubernetes assets (kubeconfig, manifests)
 module "bootkube" {
-  source = "github.com/kinvolk/terraform-render-bootkube?ref=30b2abcb1266ce9cedc92ddb3c6d8a4bbcbe7da0"
+  source = "git::https://github.com/kinvolk/terraform-render-bootkube.git?ref=mauricio/tf-0.12"
```

Comment:
The tree is probably broken before this commit. Does it make sense to you too to have the tree working at all commits and, therefore squash this with the upgrade to terraform 0.12? (maybe split by platforms if that makes it more easy to review?)

Reply:
We refer to the branch just for testing, right? BTW, you can also refer to the commit, which is not part of the master.

Before merging, that should be changed to point to the merge commit from bootkube master, so we pin it to specific version.

Reply:
I preferred to point to a branch because I can force push to https://github.com/kinvolk/terraform-render-bootkube/pull/33 without having to update the reference in this PR.

After https://github.com/kinvolk/terraform-render-bootkube/pull/33 is merged I'll update this PR to point to a single commit in the master branch.